app/src/pages/SingleContent.vue

```diff
@@ -21,10 +18,25 @@ const props = defineProps<Props>();
 const content = db.getBySlugAsRef<ContentDto>(props.slug);
 const tagsContent = ref<ContentDto[]>([]);
 
+const isExpiredOrScheduled = computed(() => {
+    if (!content.value) return false;
+    return (
+        (content.value.publishDate && content.value.publishDate > Date.now()) ||
+        (content.value.expiryDate && content.value.expiryDate < Date.now())
+    );
+});
+
 watch(
     content,
     async () => {
         if (!content.value) return;
+
+        document.title = isExpiredOrScheduled.value
+            ? `Page not found - ${appName}`
+            : `${content.value.title} - ${appName}`;
+
+        if (isExpiredOrScheduled.value) return;
```

Comment:
I think this would be clearer with an early return:

if (isExpired)
document.title = Page not found
return;

and then the rest of the code continues